lang/ilias_de.lang

```diff
@@ -41,7 +41,7 @@ common#:#pd_enable_pdfrmpostdraft_info#:#Sofern aktiviert, wird auf dem Persönl
 pd#:#block_show_pdfrmpostdraft#:#'Meine Beitragsentwürfe' anzeigen
 forum#:#forums_thread_drafts_deleted#:#Das gewählte Thema einschließlich aller dazugehörigen Beiträge und evtl. Beitragsentwürfe wurde gelöscht.
 forum#:#forums_threads_drafts_deleted#:#Die gewählten Themen einschließlich aller dazugehörigen Beiträge und evtl. Beitragsentwürfe wurden gelöscht.
-forum#:#save_draft_successfully#:#Der Entwurf wurde erfolgreich gespeichert. 
+forum#:#save_draft_successfully#:#Der Entwurf wurde erfolgreich gespeichert.
```

Comment:
We seem to need a less aggressive whitespace purging, please remove the whitespace changes in this file as well.